lib/benchee/formatters/console.ex

```diff
@@ -19,12 +19,22 @@ defmodule Benchee.Formatters.Console do
       map.flatten       781.25 KB - 1.25x memory usage
 
       **All measurements for memory usage were the same**
+
+      Reduction count statistics:
+
+      Name              average  deviation      median      99th %
+      flat_map           417.00      ±9.40      411.45      715.21
+      map.flatten        806.89     ±16.62      768.02     1170.67
```

Comment:
do reductions really vary here? Interesting, i'd have expected them to be constant on a constant input.

Reply:
No, they don't vary - this is just pulled from the runtime example (you'll notice the numbers match up there). I just used it as an example.